conda-store-server/conda_store_server/worker/tasks.py

```diff
@@ -191,6 +191,10 @@ def task_update_environment_build(self, environment_id):
 
     utils.symlink(conda_prefix, environment_prefix)
 
+    if conda_store.post_update_environment_build_hook:
+        conda_store.log.info(f"calling configured post_update_environment_build_hook...")
```

Comment:
Personally don't feel this log statement is needed. But otherwise this looks great to me. Thanks @aisis-julian :smile:

Reply:
Looks like a few black formatting issues as well

Reply:
Thanks for taking a look! I removed the log statement (which was also causing the black formatting issue)